detekt-rules-naming/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/naming/BooleanPropertyNamingSpec.kt

```diff
@@ -254,6 +254,24 @@ class BooleanPropertyNamingSpec(val env: KotlinCoreEnvironment) {
 
             assertThat(findings).isEmpty()
         }
+
+        @Test
+        fun `should highlight only the name`() {
+            val code = """
+                data class Test(
+                    /**
+                     * True if the user's e-mail address has been verified; otherwise false.
+                     */
+                    @Deprecated("Don't use this", replaceWith = ReplaceWith("email_verified"))
```

Comment:
wanted to use 
```suggestion
                    @SerializedName("email_verified")
```
but GSON is not on the classpath.